assets/airports/kdca.json

```diff
@@ -6,673 +6,1845 @@
     },
     "icao": "KDCA",
     "iata": "DCA",
-    "magnetic_north": -10.8,
-    "ctr_radius": 50,
-    "ctr_ceiling": 18000,
-    "initial_alt": 5000,
-    "position": ["N38.852222", "W77.037778"],
+    "magnetic_north": 9,
+    "ctr_radius": 80,
+    "ctr_ceiling": 7000,
+    "initial_alt": 3000,
+    "position": ["N38d51m05.20", "W077d02m15.80", "14.1ft"],
     "rr_radius_nm": 5.0,
-    "rr_center": ["N38.852222", "W77.037778"],
-    "has_terrain": false,
+    "rr_center": ["N38d51m05.20", "W077d02m15.80"],
+    "has_terrain": true,
     "wind": {
-        "angle": 350,
-        "speed": 7
+        "angle": 217,
```

Comment:
217@3 is still considered a tailwind by the sim, hence point deduction

Reply:
ah what utter bs the tutorial is then 🤦‍♂️ 

"In the lower right corner of the map is a small circle with a line. It's like a flag: the line trails in the direction the wind is blowing toward. If it’s pointing straight down, the wind is blowing from the North to the South. Aircraft must be assigned to different runways such that they always take off and land into the wind, unless the wind is less than 5 knots."

Reply:
^ The tutorial is correct and behavior is wrong. Added #883 for this.